samcli/commands/init/core/command.py

```diff
@@ -0,0 +1,116 @@
+                        RowDefinition(
+                            name=style(f"$ {ctx.command_path} --location gh:aws-samples/cookiecutter-aws-sam-python"),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                        RowDefinition(
+                            name=style(
+                                f"$ {ctx.command_path} --location "
+                                f"git+ssh://git@github.com/aws-samples/cookiecutter-aws-sam-python.git"
+                            ),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                        RowDefinition(
+                            name=style(f"$ {ctx.command_path} --location /path/to/template.zip"),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                        RowDefinition(
+                            name=style(f"$ {ctx.command_path} --location /path/to/template/directory"),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                        RowDefinition(
+                            name=style(f"$ {ctx.command_path} --location https://example.com/path/to/template.zip"),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                    ],
+                )
+
+    def format_options(self, ctx: Context, formatter: InitCommandHelpTextFormatter) -> None:  # type:ignore
+        # `ignore` is put in place here for mypy even though it is the correct behavior,
+        # as the `formatter_class` can be set in subclass of Command. If ignore is not set,
+        # mypy raises argument needs to be HelpFormatter as super class defines it.
+
+        self.format_description(formatter)
+        InitCommand.format_examples(ctx, formatter)
+
+        for option_heading, options in OPTIONS_INFO.items():
+            opts: List[RowDefinition] = sorted(
+                [
+                    CoreCommand.convert_param_to_row_definition(
+                        ctx=ctx, param=param, rank=options.get("option_names", {}).get(param.name, {}).get("rank", 0)
+                    )
+                    for param in self.get_params(ctx)
+                    if param.name in options.get("option_names", {}).keys()
+                ],
+                key=lambda row_def: row_def.rank,
+            )
+            with formatter.indented_section(name=option_heading, extra_indents=1):
+                formatter.write_rd(options.get("extras", [RowDefinition()]))
+                formatter.write_rd(
+                    [RowDefinition(name="", text="\n")]
+                    + [
+                        opt
+                        for options in zip(opts, [RowDefinition(name="", text="\n")] * (len(opts)))
+                        for opt in options
+                    ],
+                )
```

Comment:
Is there a reason for not using `CoreCommand._format_options()` here?
https://github.com/aws/aws-sam-cli/blob/develop/samcli/cli/core/command.py#L38